src/core/pipe/pipe.sort.js

```diff
@@ -0,0 +1,29 @@
+import {getValue} from '../column/column.service';
+import {orderBy} from '../services/utility';
+import {key as getKey, direction as getDirection} from '../sort/sort.service';
+import {find} from '../column/column.service';
+
+export default function (data, ctx, next) {
+	const values = [],
+		directions = [],
+		model = ctx.model,
+		dataModel = model.data(),
```

Comment:
every line should consist of const
const values
const directions..
please check this style in other files too